src/py42/sdk/queries/query_filter.py

```diff
@@ -53,46 +53,82 @@ def create_filter_group(query_filter_list, filter_clause):
 
 
 class QueryFilterStringField(object):
+    """Helper class for creating filters where the search value is a string."""
+
     _term = u"override_string_field_name"
 
     @classmethod
     def eq(cls, value):
+        """Returns a FilterGroup to find events where filter equals provided ``value``.
+
+        Args:
+            value (str): The value to match file events on.
+        """
         return create_eq_filter_group(cls._term, value)
 
     @classmethod
     def not_eq(cls, value):
+        """Returns a FilterGroup to find events where filter is not equal to provided ``value``.
```

Comment:
A lot of places, the word `filter` would roll off the tongue better if preceded by a _the_